configs/templates/skylab-dev/spack.yaml

```diff
@@ -46,7 +45,7 @@ spack:
     #- gdal@3.4.3
     #- geos@3.9.1
     - gftl-shared@1.5.0
-    - gsibec@1.0.6
+    - gsibec@1.0.7
     - hdf5@1.12.1
```

Comment:
In noticed that  hdf5, netcdf-c, netcdf-fortran, parallelio in this config and skylab-no-python-dev are on older versions that what is specified in common/packages.yaml. Is that intended, or should these versions be updated here as well.

Reply:
These should get updated! Let me do this today